language.md

```diff
@@ -396,8 +396,17 @@ Functions may use constants or previously defined functions (either built-in or
 defined in the file). Functions cannot be defined recursively. The return type
 is determined automatically.
 
+## Constants
+At the top level of a file, constants may be defined:
+
+    myval = 3 * 12;
+
+The body of constants may use functions defined in this file. However,
```

Comment:
Maybe `A constant's definition may use functions defined in this file`? I got hung up on whether constants have bodies, but maybe that's just me.

Reply:
This was probably copy pasta from somewhere. I'll fix it.